### ISSUE TITLE
Use a custom `errorClassName` for Pigeon codegen to prevent Class name clash

### ISSUE THREAD

**Issue by vanshg** [fluttercommunity/wakelock_plus#18]
By default, Pigeon creates an error class called `FlutterError`. However, if someone is consuming 2 libraries that both use Pigeon internally, this causes a class name clash on Kotlin (`Type FlutterError is defined multiple times`). 

This can be resolved by passing in `kotlinOptions: KotlinOptions(errorClassName: "YourCustomErrorClassName"),` in `PigeonOptions` in [`messages.dart`](https://github.com/fluttercommunity/wakelock_plus/blob/main/wakelock_plus/pigeons/messages.dart)

**Comment by l619534951** [comment]
The reason is that you did not specify a package name, try this: 
```dart
@ConfigurePigeon(PigeonOptions(
  dartOut: 'lib/src/activity_api.g.dart',
...
  kotlinOptions: KotlinOptions(package: "com.domain.plugin_name"),
...
))
```

**Comment by diegotori** [comment]
Addressed in version `1.2.0`.
